docs/machine-learning/tutorials/sentiment-analysis-cli.md

```diff
@@ -248,10 +242,10 @@ In this tutorial, you learned how to:
 > [!div class="checklist"]
 >
 > - Prepare your data for the selected ML task (problem to solve)
-> - Run the 'mlnet auto-train' command in the CLI tool
+> - Run the 'mlnet classification' command in the CLI tool
 > - Review the quality metric results
 > - Understand the generated C# code to run the model (Code to use in your end-user app)
-> - Explore the generated C# code that was used to train the "best quality" model (Learning purposes)
+> - Explore the generated C# code that was used to train the "best quality" model (earning purposes)
```

Comment:
Is this right? earning? If not, I'd update or remove this part

Reply:
Not sure, this bit of text was there before I think. Must be learning purposes